### PR TITLE
RD-5780 Migrated pluginsNum widget

### PR DESCRIPTION
**Pull request by Vorbert-Kruk**
## Description

Changes consists of:
- fully migrating `pluginsNum` widget to TS
- migrating away from implicit `lodash` imports
- migrating `pluginsNum` widget labels to the translation file

## Screenshots / Videos

N/A

## Checklist

- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I followed the [UI Testing Guidelines](https://cloudifysource.atlassian.net/l/cp/udCVfvrx) and verified that all tests/checks have passed.
- [x] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

## Tests

 

[Build Status](https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/4523/)

## Documentation

N/A

### CODE REVIEW

File: widgets/pluginsNum/src/widget.tsx [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
+}>;
 
-Stage.defineWidget({
-    id: 'pluginsNum',
-    name: 'Number of plugins',
-    description: 'Number of plugins',
+Stage.defineWidget<never, WidgetData, WidgetConfiguration>({
+    id: widgetId,
+    name: translate('name'),
+    description: translate('description'),
```
Comment (kubama) [inline widgets/pluginsNum/src/widget.tsx:18]: These can be now dropped

File: widgets/pluginsNum/src/widget.tsx [hunk 2]
```diff
@@ -28,21 +37,19 @@ Stage.defineWidget({
...
+        if (Stage.Utils.isEmptyWidgetData(data)) {
             return <Loading />;
         }
 
-        const { KeyIndicator } = Stage.Basic;
-        const { Link } = Stage.Shared;
-
-        const num = _.get(data, 'metadata.pagination.total', 0);
-        const to = widget.configuration.page ? `/page/${widget.configuration.page}` : '/';
+        const pluginsQuantity = data.metadata.pagination.total;
```
Comment (kubama) [inline widgets/pluginsNum/src/widget.tsx:47]: How about `pluginsCount`? I think it would be less surprising.

Comment (Vorbert-Kruk) [reply widgets/pluginsNum/src/widget.tsx:47]: Seems like a great idea 💪 
Will implement it right away 🎉

File: widgets/pluginsNum/src/widget.tsx (hunk 1 above)
Comment (Vorbert-Kruk) [reply widgets/pluginsNum/src/widget.tsx:18]: Yeah, you're right 💪 
Timing of weekly and submitting this PR wasn't the best 😄

Comment (Vorbert-Kruk) [reply widgets/pluginsNum/src/widget.tsx]: Addressed within 90a412082766fbdc4f1c621e08341b277fd3113a

File: widgets/pluginsNum/src/widget.tsx (hunk 2 above)
Comment (Vorbert-Kruk) [reply widgets/pluginsNum/src/widget.tsx]: Addressed within 90a412082766fbdc4f1c621e08341b277fd3113a

**Review by kubama — APPROVED** [review]

**Pull request merged**